### PR TITLE
Replace InitOnLoad with MRTK > Utils > WMR > Check Config

### PR DESCRIPTION
**Pull request by davidkline-ms**
This change continues the work started by @wiwei to replace InitOnLoad classes with discrete menu options.

The new Mixed Reality Toolkit > Utilities > Windows Mixed Reality > Check Configuration menu item checks to see if the Dot Net WinRT NuGet package has been imported and appropriately sets the DOTNETWINRT_PRESENT flag.

### CODE REVIEW

**Comment by davidkline-ms** [comment]
added doc updates.

File: Documentation/ReleaseNotes.md [hunk 1]
```diff
@@ -90,6 +90,7 @@ In some cases there was a tradeoff that had to be made:
 See [Leap Motion Hand Tracking Configuration](CrossPlatform/LeapMotionMRTK.md) for the extra integration step.
 For those who are using ARFoundation, there's now an additional manual step in its getting started steps.
 See [ARFoundation](CrossPlatform/UsingARFoundation.md#install-required-packages) for the new steps.
+For those who will be using [Holographic Remoting](Tools/HolographicRemoting.md) on HoloLens 2, there is now a manual step to perform.
```
Comment (wiwei) [inline Documentation/ReleaseNotes.md:93]: Nit: Add a link to the #HoloLens-2 section for HolographicRemoting.md

File: Documentation/Tools/HolographicRemoting.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ These issues are particularly relevant when running on **Unity 2019.3** or later
 > [!Note]
 > There is an issue that prevents MSBuild for Unity from functioning properly on some versions of Unity 2019. If **Mixed Reality Toolkit** > **MSBuild** > **Use MSBuild for Unity dependency resolution** does not work correctly, please refer to the [manual installation instructions](#manual-dotnetadapter-installation).
 
-The best way to check is to open Window -> Package Manager and make sure MSBuild for Unity shows up in the packages list. If it's there, assume this step succeeded. If it's not there, try running Mixed Reality Toolkit -> Utilities -> Configure Unity and repeat the steps above for running the MRTK Configurator.
+The best way to check is to open **Window** > **Package Manager** and make sure MSBuild for Unity shows up in the packages list. If it's there, assume this step succeeded. If it's not there, try running **Mixed Reality Toolkit** > **Utilities** > **Configure Unity** and repeat the steps above for running the MRTK Configurator.
```
Comment (wiwei) [inline Documentation/Tools/HolographicRemoting.md:43]: I think that we'll also want to actually add steps to the HoloLens 2 section (i.e. this is now a required step right?)

Comment (davidkline-ms) [reply Documentation/Tools/HolographicRemoting.md:43]: Correct. Will do

Comment (davidkline-ms) [reply Documentation/Tools/HolographicRemoting.md:43]: thanks for pointing this line out.... it is out of date!

File: Documentation/ReleaseNotes.md (hunk 1 above)
Comment (davidkline-ms) [reply Documentation/ReleaseNotes.md:93]: fixed in next commit

**Pull request merged**